Bootstrap & Stuff

DmACKGL opened the pull request:
-Updated Bootstrap V3 to V4 (Latest)
-Added FontAwesome
-Added SweetAlerts V2
-Added .IDEA
-Formated canvas.js
-Improved canvas.js
--Unterminated statements
--Duplicated declarations
--Line 289 was a condition instead of a comparasion
--Line 293 Changed comparison of any type of var to only the same type as the var to compare (== -> ===)
--Line 318 lineNumber++ * 1 replaced with lineNumber++
-Formatted index.html
-Improved index.html
--Script and Style Tags
--Changed Nav to NavBar
--Changed assureMail() function for use of SweetAlerts
-Formated ramos.js

DmACKGL committed:
-Update Bootstrap V3->V4 (Latest)
-Added FontAwesome
-Added .IDEA
-Formated canvas.js
-Improved canvas.js
--Unterminated statements
--Duplicated declarations
--Line 289 was a condition instead of a comparasion
--Line 293 Changed comparison of any type of var to only the same type as the var to compare (== -> ===)
--Line 318 lineNumber++ * 1 replaced with lineNumber++
-Formated index.html
-Improved index.html
--Script and Style Tags
-Formated ramos.js

etra0 commented:
Hola estimado, se agradece la intención de aportar para mejorar la malla, probé la rama para ver visualmente cuales fueron los cambios y me surgieron un par de problemas:
- Los textos de los ramos y los de bienvenida se rompieron (no se muestra nada).
- El Header no es persistente en el resto de las vistas.
- Se incluyeron varios archivos innecesarios (sobre todo la carpeta `.idea`, bastaría con agregarla al `.gitignore.`
- Ahora la alerta de contacto no hace nada más que aparecer (i.e. no agrega ninguna funcionalidad).

Saludos!

DmACKGL commented:
Hola!

1.- En local funcionaban perfectamente en PC y Celulares (ya sea LAN y/o Ngrok), igualmente probaré nuevamente para descartar problemas de Cache.

2.- Se realizó formato y mejoras solo al Index por el momento

3.- Mantener archivos como .idea pueden ayudar a mantener el formato de identacion en el caso que se ocupen los editores la suite de Intellij (ideal comentar la idea)

4.- Error mío, lo arreglaré en el siguiente commit.

Muchas gracias por a considerar ayudas por parte de externos para este proyecto universitario!

Saludos!

DmACKGL committed:
Fix for https://github.com/etra0/ramos/pull/40#issuecomment-534173844
-Corrected code [canvas.js Line 289 to = instead of ==]
-Added .idea folder to .gitignore
-Added MailTo link after Swal.Fire

DmACKGL committed:
-Fixed some typos
-Fixed Swal.Fire (ok!)

DmACKGL committed:
More improvements
-Fixed some typos
-Moved all JS files to /JS/
-Created new way to load "Carreras" on Nav
-Templates system! (for code reuse)
-Created init.js for reduce recursive code
-New way to fetch JS files in parallel and load them asynchronously
-Formatted "Prioridad" index.html
-Improved "Prioridad" index.html
--Script and Style Tags
--Changed Nav to NavBar
-Fixed .gitignore (sort of)

DmACKGL commented:
Comment https://github.com/etra0/ramos/pull/40#issuecomment-534173844 2nd point addressed (2/3)

DmACKGL committed:
-Fixed some typos (Yep, again)

DmACKGL committed:
Finally added overflow to canvas!
-Added overflow to canvas (Index.html)
-Fixed Some typos (Not my fault)

DmACKGL committed:
Whoops!
-Fixed a typo.

DmACKGL commented:
Se detecto un problema en la carga de archivos JS (No carga d3 antes de los demás scripts).
![image](https://user-images.githubusercontent.com/10565361/65555838-1368a880-df04-11e9-947a-93f267518b74.png)
Me encuentro en búsqueda de una solución.

DmACKGL committed:
Fixed loading problem on mobile
-Changed relative path to absolute path

DmACKGL committed:
Failover added.
-Added Failover in the case of the js files did not load.

DmACKGL committed:
Sorry... I guess... LOL
-Fixed typos LMAO
